Babylon/groups/powerbi/__init__.py

```diff
@@ -8,16 +8,14 @@
 
 from .commands import list_commands
 from .groups import list_groups
-from ...utils.decorators import require_platform_key
 
 
 @group()
 @pass_context
-@require_platform_key("powerbi_api_scope")
-def powerbi(ctx: Context, powerbi_api_scope: str):
+def powerbi(ctx: Context):
     """Group handling communication with PowerBI API"""
     try:
-        token = DefaultAzureCredential().get_token(powerbi_api_scope)
+        token = DefaultAzureCredential().get_token("https://analysis.windows.net/powerbi/api/.default")
```

Comment:
Why  replacing configurable value with fixed one ?

Reply:
This is not supposed to be a user configurable variable. It's a windows api url.

Reply:
But we can put it in a project config file somewhere, something like routes.py

Reply:
I moved this in another branch to have a dedicated rework